00-max.js

```diff
@@ -1,3 +1,11 @@
 export function max(array) {
-    // TODO: implement max function
+    let output;
+    let isArrayHaveContent = checkArrayContent(array);
+    isArrayHaveContent == true ?  output = Math.max.apply(Math.max,array) :  output = undefined;
+
+    return output;
```

Comment:
Do you really need to store it in a variable ? why not returning the value directly ?